src/modules/wallet/utils.ts

```diff
@@ -1,15 +1,29 @@
+import { call, select } from 'redux-saga/effects'
 import { Eth } from 'web3x-es/eth'
-import { ChainId } from '@dcl/schemas'
 import { Address } from 'web3x-es/address'
-import { ContractName, getContract } from 'decentraland-transactions'
-import { Contract, providers, utils } from 'ethers'
+import {
+  ContractData,
+  ContractName,
+  getContract,
+  sendMetaTransaction
+} from 'decentraland-transactions'
+import { Provider } from 'decentraland-connect'
+import { ChainId, getChainName } from '@dcl/schemas'
+import { PopulatedTransaction, Contract, providers, utils } from 'ethers'
 import {
   getConnectedProvider,
+  getConnectedProviderChainId,
   getConnectedProviderType,
   getNetworkProvider
 } from '../../lib/eth'
 import { getChainConfiguration } from '../../lib/chainConfiguration'
-import { Networks, Wallet } from './types'
+import { AddEthereumChainParameters, Networks, Wallet } from './types'
+import { getAddress, getChainId } from './selectors'
+
+let TRANSACTIONS_API_URL = 'https://transactions-api.decentraland.co/v1'
+export const getTransactionsApiUrl = () => TRANSACTIONS_API_URL
+export const setTransactionsApiUrl = (url: string) =>
+  (TRANSACTIONS_API_URL = url)
```

Comment:
This was left hardcoded here, we might want to get it from the configs.

Reply:
The hardcoded value is the default value, but it's set via `createWalletSaga`, here: https://github.com/decentraland/decentraland-dapps/pull/166/files#diff-3653c3a63c9cf3d4372ba903dcd32fbf1778e41d2ce8ce6d4ceaec56a0c5aa03R237